### PR TITLE
Migrate editor's guide content to metanorma.org

### LINKED ISSUE #783: Migrate Editor's Guide to metanorma.org

### PR DESCRIPTION
**Pull request by manuelfuenmayor**
Fixes #783

### CODE REVIEW

File: author/editors-guide/edition-tips.adoc [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+2. Place the cursor in the bar address, write "cmd" and press ENTER. The command line will open on that folder location.
+3. In the command line, execute the command `ren *.PNG *.png`, and that's it!
+
+== AsciiMath vs. LaTeX math
+
+Math expressions can be written in AsciiMath or LaTeX math format, which is enabled by `:stem:` document attribute. Default option can be changed from AsciiMath to LaTeX math by using `:stem: latexmath`.
+
+It is recommended to use AsciiMath whenever it is possible, especially since the most of the math expressions can be represented this way. However, some special characters require the help of LaTeX math. Examples of such characters are `\backepsilon` (∍) and `\mathbb{E}` (𝔼). 
+
+In case you are not sure whether the written expression is correct and whether it would render well in either AsciiMath or LaTeX math, you can use https://www.mathjax.org/#demo[MathJax Demo] tool to check the output.
```
Comment (ronaldtse) [inline author/editors-guide/edition-tips.adoc:32]: We use Plurimath, please replace all references to other math software with Plurimath.

File: author/editors-guide/edition-tips.adoc [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+V = \frac{1}{2} \: {\bf u}^t \:
+            \int_{surface} \: {B'}^t \: D' \: B' \: ds
+               \; {\bf u}
+----
+
+Run:
+
+[source,sh]
+----
+cat math.lxml | latexmlmath -
```
Comment (ronaldtse) [inline author/editors-guide/edition-tips.adoc:52]: Indeed we need to implement this:
* https://github.com/plurimath/plurimath/issues/204

File: author/ogc/authoring-guide/migrating-docs.adoc [hunk 3]
```diff
@@ -0,0 +1,388 @@
...
+Refer to
+https://www.metanorma.org/author/ogc/topics/requirements/[ModSpec recommendations, requirements, and permissions],
+for detailed documentation.
+
+Following are some sample cases to illustrate the use of the definition
+list encoding.
+
+==== General Requirements
+
+.General requirement sample in legacy AsciiDoc syntax
```
Comment (ronaldtse) [inline author/ogc/authoring-guide/migrating-docs.adoc:221]: We don't support this anymore, please remove this.

File: author/ogc/authoring-guide/migrating-docs.adoc [hunk 4]
```diff
@@ -0,0 +1,388 @@
...
+type:: general
+label:: /req/workflows/collection/response
+part:: A successful execution of the operation shall be reported as a response with a HTTP status code '303'.
+part:: The response shall include a 'Location' header with the URL of a collection description document corresponding to the output(s) of the workflow.
+====
+----
+
+==== Requirement Class
+
+.Requirements Class sample in legacy AsciiDoc syntax
```
Comment (ronaldtse) [inline author/ogc/authoring-guide/migrating-docs.adoc:247]: Remove.

File: author/ogc/authoring-guide/migrating-docs.adoc [hunk 5]
```diff
@@ -0,0 +1,388 @@
...
+label:: http://www.opengis.net/spec/ogcapi-processes-3/1.0/req/workflows
+subject:: Web API
+inherit:: <<OAProc-1,OGC API - Processes - Part 1: Core, Conformance Class 'core'>>
+inherit:: <<rfc2616,RFC 2616 (HTTP/1.1)>>
+====
+----
+
+==== Permissions
+
+.Permission sample in legacy AsciiDoc syntax
```
Comment (ronaldtse) [inline author/ogc/authoring-guide/migrating-docs.adoc:276]: Remove.

File: author/ogc/authoring-guide/migrating-docs.adoc [hunk 6]
```diff
@@ -0,0 +1,388 @@
...
+description:: For each UML class defined or referenced in CityGML Conceptual Model:
+part:: An Implementation Specification MAY represent that class as a null class with no attributes, associations, or definition.
+part:: An Implementation Specification MAY represent an association of the UML class with a null association.
+part:: An Implementation Specification MAY represent an attribute of the UML class with a null attribute.
+====
+----
+
+==== Recommendations
+
+.Recommendation sample in legacy AsciiDoc syntax
```
Comment (ronaldtse) [inline author/ogc/authoring-guide/migrating-docs.adoc:305]: Remove.

File: author/ogc/authoring-guide/migrating-docs.adoc [hunk 7]
```diff
@@ -0,0 +1,388 @@
...
+label:: /rec/ade/uml
+description:: In addition to meeting the requirements for a CityGML ADE, an ADE should:
+part:: The <<uml_notation_section,UML notations and stereotypes>> used in the CityGML conceptual model SHOULD be applied to corresponding model elements in an ADE.
+part:: An ADE SHOULD import and use predefined classes from external conceptual UML models such as the CityGML modules or the standardized schemas of the ISO 19100 series of International Standards.
+====
+----
+
+==== Abstract tests
+
+.Abstract test sample in legacy AsciiDoc syntax
```
Comment (ronaldtse) [inline author/ogc/authoring-guide/migrating-docs.adoc:332]: Remove.

File: author/ogc/authoring-guide/migrating-docs.adoc [hunk 8]
```diff
@@ -0,0 +1,388 @@
...
+However, it should be remembered that predefined texts are supposed
+to be compliant with the flavor authoring specifications, so the overriding
+is generally not recommended.
+
+In relation to the encoding of bibliographic references,
+some examples are given below to illustrate the options available:
+
+.Typical reference entry (without auto-fetch)
+====
+[source,asciidoc]
```
Comment (ronaldtse) [inline author/ogc/authoring-guide/migrating-docs.adoc:102]: We recommend in the order of:
1. Auto-fetch from document identifier
2. Auto-fetch from DOI or OpenLibrary
3. Citation span encoding
4. Plain text.

File: author/ogc/authoring-guide/migrating-docs.adoc (hunk 3 above)
Comment (ronaldtse) [reply author/ogc/authoring-guide/migrating-docs.adoc:221]: Ah this is just to describe how to migrate. Okay.

File: author/editors-guide/edition-tips.adoc [hunk 9]
```diff
@@ -0,0 +1,219 @@
+---
+layout: author-docs
+title: Metanorma edition tips
```
Comment (ronaldtse) [inline author/editors-guide/edition-tips.adoc:3]: "Editor tips"?

File: author/editors-guide/edition-tips.adoc (hunk 1 above)
Comment (manuelfuenmayor) [reply author/editors-guide/edition-tips.adoc:32]: @ronaldtse the indicated website (MathJax) is just to check the output of an AsciiDoc, MathML or LaTeX. We currently don't an "expression checker" in Plurimath.

File: author/editors-guide/edition-tips.adoc (hunk 2 above)
Comment (manuelfuenmayor) [reply author/editors-guide/edition-tips.adoc:52]: I've commented out the related content until we have a consolidated CLI interface.

File: author/ogc/authoring-guide/migrating-docs.adoc (hunk 8 above)
Comment (manuelfuenmayor) [reply author/ogc/authoring-guide/migrating-docs.adoc:102]: @ronaldtse I've changed the order but, the related content is not a recommendation. We just tried to showcase the options available to encode bibliographic references. For instance, references with *auto-fetch, auto-fetch disabled, plain text, user-supplied label, and numeric label. *

File: author/editors-guide/edition-tips.adoc (hunk 1 above)
Comment (ronaldtse) [reply author/editors-guide/edition-tips.adoc:32]: @manuelfuenmayor Why not use plurimath.org?

File: author/ogc/authoring-guide/migrating-docs.adoc (hunk 8 above)
Comment (ronaldtse) [reply author/ogc/authoring-guide/migrating-docs.adoc]: OK

File: author/editors-guide/edition-tips.adoc (hunk 1 above)
Comment (manuelfuenmayor) [reply author/editors-guide/edition-tips.adoc:32]: Oh, I didn't know that website existed. OK.
